docs/src/whatsnew/latest.rst

```diff
@@ -127,6 +127,16 @@ This document explains the changes made to Iris for this release
    :mod:`iris.palette` in response to a deprecation warning. Using the new
    Matplotlib API also means a ``matplotlib>=3.5`` pin. (:pull:`4998`)
 
+#. `@trexfeathers`_ altered testing to accept new Dask copying behaviour from
+   `dask/dask#9555`_ - copies of a Dask array created using ``da.from_array()``
+   will all ``compute()`` to a shared identical array. So creating a
+   :class:`~iris.cube.Cube` using ``Cube(data=da.from_array(...``, then
+   using :class:`~iris.cube.Cube` :meth:`~iris.cube.Cube.copy`,
+   will produce two :class:`~iris.cube.Cube`\s that both return an identical
+   array when requesting :class:`~iris.cube.Cube` :attr:`~iris.cube.Cube.data`.
+   We do not expect this to affect typical user workflows but please get in
+   touch if you need help. (:pull:`5041`)
```

Comment:
Although this isn't technically _our_ incompatible change, I wonder if it should go in the Incompatible Changes section for greater visibility?

Reply:
dc281890d891b797684b4348aa86ff9d1dbe9b14